sentry-backtrace/src/trim.rs

```diff
@@ -2,31 +2,29 @@ use sentry_core::protocol::{Frame, Stacktrace};
 
 use crate::utils::function_starts_with;
 
-lazy_static::lazy_static! {
-    static ref WELL_KNOWN_SYS_MODULES: Vec<&'static str> = vec![
-        "std::",
-        "core::",
-        "alloc::",
-        "backtrace::",
-        "sentry::",
-        "sentry_core::",
-        "sentry_types::",
-        // these are not modules but things like __rust_maybe_catch_panic
-        "__rust_",
-        "___rust_",
-        // these are well-known library frames
-        "anyhow::",
-        "log::",
-    ];
+const WELL_KNOWN_SYS_MODULES: &[&str] = &[
```

Comment:
nice! this could have been done independently anyway. No reason for these being `Vec`s